__tests__/app.test.js

```diff
@@ -128,6 +127,27 @@ describe("GET", () => {
         });
     });
 
+    test("status 200 : should return an array of articles which includes a comment_count property", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                comment_count: expect.any(Number),
```

Comment:
this is good, but you could have just asserted they each had the new property. eg expect(article.comment_count).toEqual(expect.any(Number)) not worth changing though